index.html

```diff
@@ -4,9 +4,57 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <title>Portfolio</title>
   </head>
   <body>
-    <h1>I'm Software Developer</h1>
+    <div class="flex-container">
+      <div class="header-area">
+        <i class="fa-solid fa-bars"></i>
+        <!-- <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul> -->
+      </div>
+      <div class="banner-text" >
+        <h1>Hi There. I'm Mudasir Sherwani</h1>
+        <h2>I'm Software Developer</h2>
+        <div>
+          <p class="banner-text-pera">I can help you build a product, feature or website Look through some of my 
+            work and experience! if you like what you see and have a project you need coded, don't hasitate to 
+            contact me.
+          </p>
+          <p class="social-icons">
+            <i class="fa-brands fa-facebook"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <i class="fa-brands fa-instagram"></i>
+            <i class="fa-brands fa-github"></i>
+            <i class="fa-brands fa-linkedin"></i>
+          </p>
+        </div>
+     </div>
+          <!-- <div class="flex-container-card">
+          <div>Project#1</div>
+          <div>Project#2</div>
+          <div>Project#3</div>
+          </div> 
+          <div class="flex-container-card">
+          <div>Project#4</div>
+          <div>Project#5</div>
+          <div>Project#6</div>
+          </div>   
+          <div> -->
+      <div class="banner-text" >
+          <h1>About me</h1>
+          <p class="text-about">I can help you build a product, feature or website Look through some of my 
+            work and experience! if you like what you see and have a project you need coded, don't hasitate to 
+            contact me.
+          </p>
+          <button type="button" class="about-me">Get My Resume</button>
+      </div>
+      <div></div>
+      <div></div>
+    </div>
```

Comment:
- [ ]  Please resemble Your designs, and fonts margin with the one provided by Figma design 

Your design          |  Figma design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/43877689/176873843-362cdac2-3443-480d-bc91-2e4d3f75ad9a.png)  |  ![image](https://user-images.githubusercontent.com/43877689/176874377-315f55af-480a-424d-b120-cd82c49fedf5.png)